syntax = "proto3";

message Products {
    string product_name = 1;
    int32 product_price = 2;
    int32 product_quantity = 3;
    string product_category = 4;
}

message ProductImages {
    string image_url = 1;
    string image_name = 2;
    int32 product_id = 3; 
}
